screeps-defaults.toml

```diff
@@ -11,8 +11,9 @@ branch = "default"
 # port = 443
 
 # [copy]
-# destination = "your copy destination"
+# destination = "your copy destination"  # On Windows, use forward slashes instead of backslashes
```

Comment:
I think a comment above the line could be slightly more consistent with others in the file, but it's not a big deal!